.github/workflows/server-ci-template.yml

```diff
@@ -9,289 +9,11 @@ env:
   go-version: "1.19.5"
 
 jobs:
-  check-mocks:
```

Comment:
These removals are just to make iterating on the CI changes faster, and will be reverted before opening up the PR for final review.